soc/arm/nordic_nrf/nrf52/Kconfig.soc

```diff
@@ -412,6 +412,7 @@ config SOC_NRF52840
 	select HAS_HW_NRF_UARTE1
 	select HAS_HW_NRF_USBD
 	select HAS_HW_NRF_WDT
+	select HAS_SWO
```

Comment:
does this apply to the other nRF variants @aurel32 ?

Reply:
Hmm good catch. I added it here because it was the SoC I was using, but indeed it's better to enable it for the whole SoC series. Looking a that I realized that this has been added recently directly to the `SOC_SERIES_NRF*` symbol in this commit: https://github.com/zephyrproject-rtos/zephyr/commit/cc7a32449381510134d671ea2bad2fe33948d945 I haven't noticed that when rebasing my patch before submitting it.

That said the other change to modules/hal_nordic/nrfx/CMakeLists.txt is still needed. I'll update my PR.